src/js/modules/supervisors/HandleError.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  DisableCoprosReply,
+  EnableCoprosReply,
+} from "../../modules/domain/generatedRpc/enableDisableCoprocClasses";
+import { Coprocessor } from "../../modules/public/Coprocessor";
+
+export enum ResponseEnableCode {
+  success,
+  internalError,
+  invalidIngestionPolicy,
+  scriptIdAlreadyExist,
+  topicDoesNotExist,
+  invalidTopic,
+  materializedTopic,
+}
+
+export enum ResponseDisableCode {
+  success,
+  internalError,
+  scriptDoesNotExist,
+}
+
+export const validateDisableCodeResponse = (
+  responses: DisableCoprosReply,
+  coprocessor: Coprocessor[]
+): Error[] => {
+  return responses.inputs.reduceRight<Error[]>((errors, code, index) => {
+    switch (code) {
+      case ResponseDisableCode.internalError: {
+        errors.push(
+          new Error(
+            `internal error: wasm id function: ` + coprocessor[index].globalId
+          )
+        );
+        break;
+      }
+      case ResponseDisableCode.scriptDoesNotExist: {
+        errors.push(
+          new Error(
+            `error: script id hasn't registered, in no possible deregister: ` +
```

Comment:
Same here.

Reply:
Also, let's change the message to
`error: script with ID ${coprocessor[index].globalId} doesn't exist.`

Reply:
Done